plugins/source/aws/resources/services/cloudwatchlogs/subscription_filters.go

```diff
@@ -0,0 +1,39 @@
+package cloudwatchlogs
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func subscriptionFilters() *schema.Table {
+	return &schema.Table{
+		Name:        "aws_cloudwatchlogs_log_group_subscription_filters",
+		Description: `https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_SubscriptionFilter.html`,
+		Resolver:    fetchCloudwatchlogsSubscriptionFilters,
+		Multiplex:   client.ServiceAccountRegionMultiplexer("logs"),
+		Transform:   transformers.TransformWithStruct(&types.SubscriptionFilter{}, transformers.WithPrimaryKeys("FilterName", "CreationTime")),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(true),
+			client.DefaultRegionColumn(true),
+		},
+	}
+}
+func fetchCloudwatchlogsSubscriptionFilters(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	var config cloudwatchlogs.DescribeSubscriptionFiltersInput
```

Comment:
Not using a log group from the parent as input, so this resource should be on the top level?